config.rst

```diff
@@ -473,8 +475,8 @@ tools:
 - `Symfony CLI <https://symfony.com/download>`_
 - `Composer <https://getcomposer.org/download/>`_
 
-Additionally, it creates four helpers: symfony-build_, symfony-deploy_,
-symfony-database-migrate_ and yarn-install_.
+Additionally, it creates five helpers: symfony-build_, symfony-deploy_,
```

Comment:
it creates some helpers?

Reply:
It creates the following helpers....

Removing them would be bad when someone searchs for this.

Reply:
I think Fabien only wanted me to replace the number (to avoid updating it each time).
At least, this is what I did 😃

Reply:
Yep :)

Reply:
👍